packages/anoncreds/src/services/AnonCredsHolderServiceOptions.ts

```diff
@@ -81,9 +81,19 @@ export type GetCredentialsForProofRequestReturn = Array<{
 export interface CreateCredentialRequestOptions {
   credentialOffer: AnonCredsCredentialOffer
   credentialDefinition: AnonCredsCredentialDefinition
+  masterSecretId?: string
```

Comment:
```suggestion
  linkSecretId?: string
```

Reply:
I've applied the suggestion. It's currently not used in the IndySdkHolderService (it's using the masterSecretId from the wallet).

However, I think we can best address this in #1198